### PR TITLE
Changed image of openid-connect logo for better look on arc-green theme

### PR DESCRIPTION
**Pull request by leona-ya**
When I use the arc-green theme and enable the openID Connect authentication source I experience the bug that the logo of OIDC has dark text on dark background. I changed the background of the logo to white.

### CODE REVIEW

**Comment by a1012112796** [comment]
Please fix lint, suggest add screenshoots about this change , Thanks

File: web_src/less/themes/theme-arc-green.less [hunk 1]
```diff
@@ -1610,3 +1610,11 @@ footer .container .links > * {
...
     background-color: #888;
 }
+
+.signin {
+    .oauth2 {
+        img.openidConnect {
+                background-color: #fff;
+        }
+    }
+}
```
Comment (a1012112796) [inline web_src/less/themes/theme-arc-green.less]:
```suggestion
.signin .oauth2 img.openidConnect {
    background-color: #fff;
}
```
I think this style is more simple, and the result is same with what you want

Comment (silverwind) [reply web_src/less/themes/theme-arc-green.less]: Yes, no need to get fancy with Less nesting unless it makes sense.

**Comment by em0lar** [comment]
Before the change:
image

After the change:
image

File: web_src/less/themes/theme-arc-green.less (hunk 1 above)
Comment (leona-ya) [reply web_src/less/themes/theme-arc-green.less]: Thank you, I changed this. Below I added some pictures

**Review by zeripath — APPROVED** [review]
As per their style guidelines it should be on a white background or we should switch to the monochrome white on black logo

**Comment by silverwind** [comment]
There is also https://openid.net/openid_connect_new-logo/ which has colors that work on both themes but it has to be converted to SVG first.

**Comment by zeripath** [comment]
I've had a go at SVG'ing that logo - here's the svg in a zip.


[OPENID_CONNECT_NEW-Logo.zip](https://github.com/go-gitea/gitea/files/4598861/OPENID_CONNECT_NEW-Logo.zip)

**Comment by em0lar** [comment]
Here i took a screenshot of both themes with the new logo. I can push this changes, if you like them more than my initial idea
image
image

**Comment by silverwind** [comment]
Looking great there, let's replace the image.

**Comment by silverwind** [comment]
I optimized the SVG further, can you use this one?

```svg
<svg xmlns="http://www.w3.org/2000/svg" width="6095" height="2826"><path fill="#636363" d="M3454 2822c-49-3-91-16-124-39a191 191 0 01-83-138c-2-12-2-42 0-55a193 193 0 0174-130c40-31 90-46 154-46 28 0 52 3 77 9l58 15-3 9c-5 15-9 37-11 61l-1 12c-1 1-11 2-14 1-2-1-2-3-2-17 0-23-2-32-13-43-17-18-46-26-94-26-23 0-35 1-53 5-43 9-74 34-95 76-12 24-17 48-19 82-1 37 6 72 20 101 8 17 17 29 32 43 29 29 64 45 108 50 16 2 54 1 69-2 24-4 46-12 65-24l8-5 1 2 2 4-5 12c-5 14-5 13-24 21a272 272 0 01-127 22zm297 0a118 118 0 01-66-32c-25-25-38-60-38-102 0-18 1-28 5-42 12-47 50-84 98-94 35-8 71-5 99 9 45 23 70 68 68 124-2 39-17 73-44 98a151 151 0 01-122 39zm60-21c19-7 36-22 44-43 8-19 11-45 9-75-2-36-11-63-26-85a75 75 0 00-63-33c-14 0-23 2-34 8-11 5-25 18-30 29-9 17-12 30-13 58 0 29 3 50 11 75 10 33 27 53 51 63 16 7 35 8 51 3zm914 19c-52-5-90-32-106-78-6-18-7-27-7-53 0-22 0-25 2-37 5-21 14-42 23-54 11-15 42-35 66-43 14-5 25-7 42-7 30 0 54 9 74 28 12 12 20 26 25 44 4 15 5 38 3 49-1 4-1 4-17 7-21 3-40 4-103 4l-63 1v10a102 102 0 0031 77c19 18 51 28 79 24 15-1 25-5 45-15 18-9 18-9 19-7 2 2 2 3-1 8l-4 10c-1 3-3 4-10 8a154.1 154.1 0 01-98 24zm64-158c11-1 11-1 9-22-2-29-7-44-20-57a57 57 0 00-45-18c-20 0-33 6-46 19-9 9-14 17-18 30-3 10-6 29-6 38l1 7 4 1c9 1 103 3 121 2zm213 158c-51-4-88-32-104-77-11-33-10-79 2-110 11-29 24-41 65-64 20-11 29-15 41-18 9-3 11-3 33-3 24 0 32 1 47 5l20 10c1 1 0 5-1 18a718 718 0 00-6 52l-7 1c-8 0-7 2-9-18l-3-17c-3-7-18-16-37-21-8-3-10-3-26-3-19 0-27 1-39 7-21 11-34 34-38 66-1 12-1 37 2 49 8 48 31 79 66 91 25 8 51 7 79-5 14-6 16-6 18-2l2 3-5 8-7 10-31 13c-17 5-43 7-62 5zm215-3c-22-3-37-18-42-39v-179l-1-3-20 1c-23 0-22 0-22-6 0-3 2-5 9-8 12-4 32-14 33-15l1-43v-41l9-4c5-1 15-6 23-10 11-5 14-7 16-6 4 2 4 3 3 19l-5 78v9l39-1h38c1 1 0 6-2 13l-2 12h-36c-27 0-36 1-37 2l-1 82v82l2 6c7 14 19 20 41 20 9 0 21-2 27-4 2-1 2 0 3 2 3 7 3 6-10 17l-15 12c-10 4-38 6-51 4zm-1271-4l-1-7 1-6 15-1c21 0 24-2 27-16 2-11 2-178 0-186s-4-9-25-10h-17v-15l11-2c24-3 47-9 62-15 14-6 15-6 17-4l1 7a552 552 0 00-1 41l27-21c24-21 25-21 32-23 18-6 39-7 55-4 18 4 35 12 46 23 9 9 15 22 18 35 2 9 2 14 3 88 1 79 1 87 5 94 3 7 7 8 24 8l16 1v14h-4a824 824 0 00-90-1l1-33c2-44 1-142-1-153-5-21-23-37-47-42-10-2-27-2-36 1-21 6-41 22-46 38-4 13-4 21-4 87 1 63 2 74 5 81 3 6 6 7 24 7 10 0 16 1 16 2v12h-21a1546 1546 0 00-113 0zm329 0v-12c0-1 3-2 15-2 13 0 15 0 19-2 5-3 8-6 9-16 2-9 3-151 1-170-1-23-2-24-27-24l-17-1v-13l8-2a322 322 0 0081-22c2 0 3 4 2 17v33c1 0 13-9 26-21 24-20 25-21 32-23a112 112 0 0169 0c30 10 47 29 52 60 1 4 2 34 2 83 1 79 1 88 5 94 3 6 7 7 24 7l16 1v14h-4a819 819 0 00-91 0l2-37c2-48 1-138-1-149-4-16-14-29-27-36-12-6-20-8-35-8-17 0-23 1-36 7-15 8-27 20-31 33-4 12-4 19-4 86 1 65 2 75 5 81 3 7 7 8 24 8l16 1v14l-16-1a2243 2243 0 00-119 0z"/><path fill="#f6921e" d="M3321 2460c-6862 990-3431 495 0 0zm-560 285c-1.4-327-1.1-654 1-981 38-1.2 76-.8 114 1l1 48c0 52 0 51 6 40 14-26 37-50 62-65 29-19 72-33 118-38 19-3 68-3 88 0 65 8 116 30 164 70 70 61 114 165 118 280 5 153-46 275-143 339-43 29-90 44-147 49-20 2-62 1-79-2-50-7-89-21-122-42-25-17-43-34-57-56l-7-8-1 181-1 183-57 1zm367-359c38-4 67-16 95-39 48-38 76-100 85-188 2-21 2-65 0-82-9-84-40-147-91-186-27-21-58-34-96-39h-63a188 188 0 00-134 80c-34 48-51 110-51 187 0 122 44 209 126 249 23 11 47 17 82 20zm-984 109a431 431 0 01-285-127 528 528 0 01-141-335c-2-19-2-78 0-98 11-138 61-255 143-338a433 433 0 01325-127 432 432 0 01369 177 601 601 0 01-12 689 435 435 0 01-297 158c-18 2-85 3-102 1zm66-108a300 300 0 00258-161 530 530 0 0032-410c-25-80-71-143-130-184-44-29-86-44-140-50-50-6-104-1-148 12-48 15-98 47-133 85-39 44-68 101-84 164-13 51-17 86-17 141a489 489 0 0049 227c33 65 79 113 139 144 50 25 108 36 174 32zm-987-1043l-1-1145 194-97L1630-2v2302c-1 2-405 193-408 193zm2579 1144c-45-5-81-14-115-31-56-27-99-69-130-124-24-43-41-101-48-166-11-93 5-180 46-253a328 328 0 01242-164c18-3 67-3 87 0 64 8 109 27 156 69 26 22 47 50 66 82a440 440 0 0154 252l-266 1h-265v15c0 45 13 91 34 125 32 51 85 83 151 92 16 2 53 2 68 0 35-4 61-13 86-30a163 163 0 0063-93l4-13h57l57-1v5l-11 43c-15 43-35 78-63 107-45 46-102 73-177 83-17 2-78 3-96 1zm234-444c0-16-7-44-15-65-21-54-61-94-115-116-42-16-98-17-142-2-15 6-36 16-49 25-11 8-28 24-37 35a231 231 0 00-49 129l204 1h204zm205 75l1-355h108l1 57 1 55c1 0 3-2 9-12 12-19 30-41 45-54 18-16 34-27 55-37 39-19 82-27 135-25 52 1 90 9 124 27a175 175 0 0180 82c14 26 21 51 26 94 2 17 2 17 3 270v253h-116l-1-253v-252l-3-11c-6-30-18-52-37-70-27-27-66-39-115-37-36 1-65 8-91 20-58 28-95 83-105 158-3 21-4 51-4 248v197h-116zm748-135v-491l72 1h72v980h-144zm296 489l-1-490 1-489h383c157 9 277 66 347 166 39 57 64 130 73 217 4 36 5 41 5 86-1 45-1 64-5 100-18 160-75 269-175 336a514 514 0 01-244 74c-25 2-384 2-384 0zm385-110c62-5 109-22 159-55 43-29 78-75 100-133 18-45 26-84 31-140 2-19 2-94 0-114-4-51-11-90-22-124a263 263 0 00-233-190c-29-3-40-4-166-4h-124v761h119z"/><path fill="#babbbd" d="M1149 753a1672 1672 0 00-842 317 917 917 0 00-215 232 610 610 0 00-92 367c11 114 54 220 129 321 138 183 376 334 670 422a2101 2101 0 00423 81v-257l-24-3c-168-24-324-74-451-144a786 786 0 01-236-192c-45-57-77-119-92-181-16-63-16-128-1-191 17-69 50-133 103-196 16-20 59-62 81-81a927 927 0 01230-141 1354 1354 0 01316-94l3-1V883c.2-43.3-.1-86.7-1-130zm555 0h-1l-1 130 1 129 6 1 26 5c98 19 194 48 280 86 37 16 107 52 107 55l-108 62a7663.5 7663.5 0 00-109 63l97 21 689 150-56-583a5534 5534 0 00-208 115l-17-9a1836 1836 0 00-706-225z"/></svg>

**Comment by em0lar** [comment]
Thanks for your work. I updated the image. Now (for documentation the following images) it looks like this:
image

image

**Review by zeripath — APPROVED** [review]

**Comment by silverwind** [comment]
Sorry for being so picky, but can you remove the whitespace in the SVG source? It should be a single line in the editor.

File: public/img/auth/openid_connect.svg [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    <path fill="#babbbd"
+          d="M1149 753a1672 1672 0 00-842 317 917 917 0 00-215 232 610 610 0 00-92 367c11 114 54 220 129 321 138 183 376 334 670 422a2101 2101 0 00423 81v-257l-24-3c-168-24-324-74-451-144a786 786 0 01-236-192c-45-57-77-119-92-181-16-63-16-128-1-191 17-69 50-133 103-196 16-20 59-62 81-81a927 927 0 01230-141 1354 1354 0 01316-94l3-1V883c.2-43.3-.1-86.7-1-130zm555 0h-1l-1 130 1 129 6 1 26 5c98 19 194 48 280 86 37 16 107 52 107 55l-108 62a7663.5 7663.5 0 00-109 63l97 21 689 150-56-583a5534 5534 0 00-208 115l-17-9a1836 1836 0 00-706-225z"/>
+</svg>
```
Comment (silverwind) [inline public/img/auth/openid_connect.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="6095" height="2826"><path fill="#636363" d="M3454 2822c-49-3-91-16-124-39a191 191 0 01-83-138c-2-12-2-42 0-55a193 193 0 0174-130c40-31 90-46 154-46 28 0 52 3 77 9l58 15-3 9c-5 15-9 37-11 61l-1 12c-1 1-11 2-14 1-2-1-2-3-2-17 0-23-2-32-13-43-17-18-46-26-94-26-23 0-35 1-53 5-43 9-74 34-95 76-12 24-17 48-19 82-1 37 6 72 20 101 8 17 17 29 32 43 29 29 64 45 108 50 16 2 54 1 69-2 24-4 46-12 65-24l8-5 1 2 2 4-5 12c-5 14-5 13-24 21a272 272 0 01-127 22zm297 0a118 118 0 01-66-32c-25-25-38-60-38-102 0-18 1-28 5-42 12-47 50-84 98-94 35-8 71-5 99 9 45 23 70 68 68 124-2 39-17 73-44 98a151 151 0 01-122 39zm60-21c19-7 36-22 44-43 8-19 11-45 9-75-2-36-11-63-26-85a75 75 0 00-63-33c-14 0-23 2-34 8-11 5-25 18-30 29-9 17-12 30-13 58 0 29 3 50 11 75 10 33 27 53 51 63 16 7 35 8 51 3zm914 19c-52-5-90-32-106-78-6-18-7-27-7-53 0-22 0-25 2-37 5-21 14-42 23-54 11-15 42-35 66-43 14-5 25-7 42-7 30 0 54 9 74 28 12 12 20 26 25 44 4 15 5 38 3 49-1 4-1 4-17 7-21 3-40 4-103 4l-63 1v10a102 102 0 0031 77c19 18 51 28 79 24 15-1 25-5 45-15 18-9 18-9 19-7 2 2 2 3-1 8l-4 10c-1 3-3 4-10 8a154.1 154.1 0 01-98 24zm64-158c11-1 11-1 9-22-2-29-7-44-20-57a57 57 0 00-45-18c-20 0-33 6-46 19-9 9-14 17-18 30-3 10-6 29-6 38l1 7 4 1c9 1 103 3 121 2zm213 158c-51-4-88-32-104-77-11-33-10-79 2-110 11-29 24-41 65-64 20-11 29-15 41-18 9-3 11-3 33-3 24 0 32 1 47 5l20 10c1 1 0 5-1 18a718 718 0 00-6 52l-7 1c-8 0-7 2-9-18l-3-17c-3-7-18-16-37-21-8-3-10-3-26-3-19 0-27 1-39 7-21 11-34 34-38 66-1 12-1 37 2 49 8 48 31 79 66 91 25 8 51 7 79-5 14-6 16-6 18-2l2 3-5 8-7 10-31 13c-17 5-43 7-62 5zm215-3c-22-3-37-18-42-39v-179l-1-3-20 1c-23 0-22 0-22-6 0-3 2-5 9-8 12-4 32-14 33-15l1-43v-41l9-4c5-1 15-6 23-10 11-5 14-7 16-6 4 2 4 3 3 19l-5 78v9l39-1h38c1 1 0 6-2 13l-2 12h-36c-27 0-36 1-37 2l-1 82v82l2 6c7 14 19 20 41 20 9 0 21-2 27-4 2-1 2 0 3 2 3 7 3 6-10 17l-15 12c-10 4-38 6-51 4zm-1271-4l-1-7 1-6 15-1c21 0 24-2 27-16 2-11 2-178 0-186s-4-9-25-10h-17v-15l11-2c24-3 47-9 62-15 14-6 15-6 17-4l1 7a552 552 0 00-1 41l27-21c24-21 25-21 32-23 18-6 39-7 55-4 18 4 35 12 46 23 9 9 15 22 18 35 2 9 2 14 3 88 1 79 1 87 5 94 3 7 7 8 24 8l16 1v14h-4a824 824 0 00-90-1l1-33c2-44 1-142-1-153-5-21-23-37-47-42-10-2-27-2-36 1-21 6-41 22-46 38-4 13-4 21-4 87 1 63 2 74 5 81 3 6 6 7 24 7 10 0 16 1 16 2v12h-21a1546 1546 0 00-113 0zm329 0v-12c0-1 3-2 15-2 13 0 15 0 19-2 5-3 8-6 9-16 2-9 3-151 1-170-1-23-2-24-27-24l-17-1v-13l8-2a322 322 0 0081-22c2 0 3 4 2 17v33c1 0 13-9 26-21 24-20 25-21 32-23a112 112 0 0169 0c30 10 47 29 52 60 1 4 2 34 2 83 1 79 1 88 5 94 3 6 7 7 24 7l16 1v14h-4a819 819 0 00-91 0l2-37c2-48 1-138-1-149-4-16-14-29-27-36-12-6-20-8-35-8-17 0-23 1-36 7-15 8-27 20-31 33-4 12-4 19-4 86 1 65 2 75 5 81 3 7 7 8 24 8l16 1v14l-16-1a2243 2243 0 00-119 0z"/><path fill="#f6921e" d="M3321 2460c-6862 990-3431 495 0 0zm-560 285c-1.4-327-1.1-654 1-981 38-1.2 76-.8 114 1l1 48c0 52 0 51 6 40 14-26 37-50 62-65 29-19 72-33 118-38 19-3 68-3 88 0 65 8 116 30 164 70 70 61 114 165 118 280 5 153-46 275-143 339-43 29-90 44-147 49-20 2-62 1-79-2-50-7-89-21-122-42-25-17-43-34-57-56l-7-8-1 181-1 183-57 1zm367-359c38-4 67-16 95-39 48-38 76-100 85-188 2-21 2-65 0-82-9-84-40-147-91-186-27-21-58-34-96-39h-63a188 188 0 00-134 80c-34 48-51 110-51 187 0 122 44 209 126 249 23 11 47 17 82 20zm-984 109a431 431 0 01-285-127 528 528 0 01-141-335c-2-19-2-78 0-98 11-138 61-255 143-338a433 433 0 01325-127 432 432 0 01369 177 601 601 0 01-12 689 435 435 0 01-297 158c-18 2-85 3-102 1zm66-108a300 300 0 00258-161 530 530 0 0032-410c-25-80-71-143-130-184-44-29-86-44-140-50-50-6-104-1-148 12-48 15-98 47-133 85-39 44-68 101-84 164-13 51-17 86-17 141a489 489 0 0049 227c33 65 79 113 139 144 50 25 108 36 174 32zm-987-1043l-1-1145 194-97L1630-2v2302c-1 2-405 193-408 193zm2579 1144c-45-5-81-14-115-31-56-27-99-69-130-124-24-43-41-101-48-166-11-93 5-180 46-253a328 328 0 01242-164c18-3 67-3 87 0 64 8 109 27 156 69 26 22 47 50 66 82a440 440 0 0154 252l-266 1h-265v15c0 45 13 91 34 125 32 51 85 83 151 92 16 2 53 2 68 0 35-4 61-13 86-30a163 163 0 0063-93l4-13h57l57-1v5l-11 43c-15 43-35 78-63 107-45 46-102 73-177 83-17 2-78 3-96 1zm234-444c0-16-7-44-15-65-21-54-61-94-115-116-42-16-98-17-142-2-15 6-36 16-49 25-11 8-28 24-37 35a231 231 0 00-49 129l204 1h204zm205 75l1-355h108l1 57 1 55c1 0 3-2 9-12 12-19 30-41 45-54 18-16 34-27 55-37 39-19 82-27 135-25 52 1 90 9 124 27a175 175 0 0180 82c14 26 21 51 26 94 2 17 2 17 3 270v253h-116l-1-253v-252l-3-11c-6-30-18-52-37-70-27-27-66-39-115-37-36 1-65 8-91 20-58 28-95 83-105 158-3 21-4 51-4 248v197h-116zm748-135v-491l72 1h72v980h-144zm296 489l-1-490 1-489h383c157 9 277 66 347 166 39 57 64 130 73 217 4 36 5 41 5 86-1 45-1 64-5 100-18 160-75 269-175 336a514 514 0 01-244 74c-25 2-384 2-384 0zm385-110c62-5 109-22 159-55 43-29 78-75 100-133 18-45 26-84 31-140 2-19 2-94 0-114-4-51-11-90-22-124a263 263 0 00-233-190c-29-3-40-4-166-4h-124v761h119z"/><path fill="#babbbd" d="M1149 753a1672 1672 0 00-842 317 917 917 0 00-215 232 610 610 0 00-92 367c11 114 54 220 129 321 138 183 376 334 670 422a2101 2101 0 00423 81v-257l-24-3c-168-24-324-74-451-144a786 786 0 01-236-192c-45-57-77-119-92-181-16-63-16-128-1-191 17-69 50-133 103-196 16-20 59-62 81-81a927 927 0 01230-141 1354 1354 0 01316-94l3-1V883c.2-43.3-.1-86.7-1-130zm555 0h-1l-1 130 1 129 6 1 26 5c98 19 194 48 280 86 37 16 107 52 107 55l-108 62a7663.5 7663.5 0 00-109 63l97 21 689 150-56-583a5534 5534 0 00-208 115l-17-9a1836 1836 0 00-706-225z"/></svg>
```

**Review by silverwind — APPROVED** [review]

**Comment by em0lar** [comment]
sorry, i clicked on a wrong button

**Comment by a1012112796** [comment]
Need restart ci for this PR, Thanks

**Review by lafriks — APPROVED** [review]

**Pull request merged**